docs/06-documentation-and-release-notes/files/README-for-Demos.md

```diff
@@ -0,0 +1,130 @@
+# README.md Template for Demos
+
+# [Connector_Name] Anypoint&trade; Studio Connector Demo - [Demo_Name]
```

Comment:
Yeah, I would remove the "Anypoint" too. I agree that syntaxis you suggest, @martinpaoloni.